qutip/bloch.py

```diff
@@ -288,6 +294,9 @@ def clear(self):
         self.vectors = []
         self.point_style = []
         self.annotations = []
+        self._lines = []
+        self._arcs = []
+        self.arc_style = []
```

Comment:
Is this not being used anywhere.

Reply:
It is. We fill the _lines and _arcs when adding lines and adding arcs. Then, it is being used in `plot_line` and `plot_arc`.